mongo/datadog_checks/mongo/config_models/instance.py

```diff
@@ -58,7 +58,7 @@ class Config:
     database: Optional[str]
     disable_generic_tags: Optional[bool]
     empty_default_hostname: Optional[bool]
-    hosts: Optional[Sequence[str]]
+    hosts: Optional[Union[str, Sequence[str]]]
```

Comment:
This file is part of the config models autogenerated from the spec. Today it is only used on python3 (Agent 7) to raise errors if the input is of the wrong type